src/main/telemetry/crsf.c

```diff
@@ -414,7 +408,7 @@ void crsfFrameFlightMode(sbuf_t *dst)
 
     sbufWriteString(dst, flightMode);
     if (!ARMING_FLAG(ARMED)) {
-        sbufWriteU8(dst, '*');
+        sbufWriteU8(dst, isArmingDisabled() ? '!' : '*');
```

Comment:
? for gps nor ready?

Reply:
Please read initial Feature Request.
It was ditched in favour of '!' marker not to obscure real flight mode.
So before arming you always see what flight mode is selected.